WireUI/Sources/WireAccountImageUI/AccountImageGenerator/AccountImageGenerator.swift

```diff
@@ -61,6 +61,7 @@ private final class InitialsRenderer: UIView {
 
     private func setupLabel(_ initials: String) {
         initialsLabel.font = .systemFont(ofSize: 13, weight: .light)
+        initialsLabel.textColor = .black
```

Comment:
Setting it to black means when switching to dark mode the label's color would still be black. Is this actually the desired behaviour?

Reply:
Good point, I will revert this and ask in the design team.